### ISSUE TITLE
Get data repo on Travis machine, so shot data tests can pass

### ISSUE THREAD

**Issue by aenfield** [awfuldynne/golf_course_project#14]
Looks like the right way to do a dependency on another repo is via 'submodule' functionality in Git, and doing it this way may make it easy/possible for us to rely on the data repo from Travis CI. I need to investigate how submodules work and if needed go ahead and implement the organization in our project.

**Comment by aenfield** [comment]
Updating the title/description to focus on what we need - getting the data repo on the Travis machine - and not on one possible implementation detail (git submodules).

**Comment by aenfield** [comment]
This works (!) - I figured out Git submodules, Travis encryption/decryption, and Git deploy keys, and updated .travis.yml - finishing with 4141ea4754efcf12497278acceeb928cdc29fd6f - so that the CI build a) defers submodule download, b) decrypts and adds the required deploy private key (which matches the deploy public key I added to the newly slimmed data repo), c) pulls the data repo into the submodule, d) unzips the zip files, and e) runs tests. I removed the pylint run for now at least so I could get the tests running.

22 of 23 tests pass - I added a new issue for Sean to fix his weather test, after I tried and didn't fix it.
